tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -247,13 +250,22 @@ void empty_braces_helper(
 
 template <typename CharT>
 void test_duration_formatter() {
+    using LongRatio = ratio<intmax_max - 1, intmax_max>;

```

Comment:
Surprisingly (and pre-existingly), should include `<ratio>` (`<chrono>` isn't guaranteed to drag it in).